src/mappings_test.go

```diff
@@ -418,8 +432,11 @@ func TestLinkSourceNotExist(t *testing.T) {
 	}
 }
 
-func TestLinkNullDist(t *testing.T) {
-	m := Mappings{"License.txt": abspath.AbsPath{}}
+func TestLinkNullDest(t *testing.T) {
+	m := Mappings{
+		"empty":     []abspath.AbsPath{},
+		"null_only": []abspath.AbsPath{abspath.AbsPath{}},
+	}
```

Comment:
Please add test case for converting `MappingsJSON` to `Mappings` with these edge cases. It will catch mistakes such as https://github.com/rhysd/dotfiles/pull/7#discussion_r467727679.

Reply:
I added this case.
https://github.com/rhysd/dotfiles/pull/7/files#diff-efaf99ab42d0924fac6876b60948027cR589-R606